OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseOutlineFragment.java

```diff
@@ -296,7 +302,9 @@ private void removeCalendar(Long calendarId) {
     }
 
     private void trackCalendarEvent(String eventName, String biValue) {
-        environment.getAnalyticsRegistry().trackCalendarEvent(eventName, biValue, courseData.getCourseId(), courseData.getMode(), courseData.getCourse().isSelfPaced());
+        environment.getAnalyticsRegistry().trackCalendarEvent(eventName, biValue, courseData.getCourseId(),
+                courseData.getMode(), courseData.getCourse().isSelfPaced(), courseDateViewModel.getEventCreateStartTime());
+        courseDateViewModel.setEventCreateStartTime(0L);
```

Comment:
I think a `resetCalendarEventTime()` function in viewModel might be a better alternative.